src/save_index.cpp

```diff
@@ -46,6 +46,16 @@ void replace_space2underbar(std::string &name) {
 	std::replace(name.begin(),name.end(),' ','_');
 }
 
+/** Replace the spaces in the filter */
+std::string replace_filter(const std::string* filter)
```

Comment:
My intention was to comply with the same guidelines of the functions **replace_underbar2space** and **replace_space2underbar**.